v3/routes/tracks.rb

```diff
@@ -8,6 +8,12 @@ class Tracks < Core
       get '/tracks/:id' do |id|
         pg :track, locals: { track: config.find(id) }
       end
+
+      get '/tracks/:id/:exercise_slug/readme' do |id, slug|
+        problem = config.find(id).find(slug)
+        problem.readme? or halt 404, { error: problem.error }.to_json
```

Comment:
We have track-level linting so we will never deploy a problem that doesn't have metadata.

We can assume that the README always exists.